README.md

```diff
@@ -33,6 +35,28 @@ In development you can quickly get an environment running using Docker by follow
    else should work well for development.
 1. Run `make up`. This will download dependencies, build and launch a development version of Outline.
 
+For production deployment by building from scratch, you can follow these steps:
+
+1. Clone this repo and install dependencies with `npm install` or `yarn`
```

Comment:
I think the installation guide should prefer `yarn`, as we use yarn for development and a yarn lockfile is present there's a small possibility that installing with NPM will result in a different set of dependencies being installed

Reply:
Fixed